src/main/java/org/example/controller/DeveloperController.java

```diff
@@ -62,11 +66,34 @@ public Response getJsonType(Arguments arguments) {
         return response;
     }
 
+    public Response create(Arguments arguments) {
+        Response response = new Response();
+
+        Developer developer = argumentToDeveloperConverter.convert(arguments);
+        String jsonString = new Gson().toJson(developer);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        try {
+            Developer newDeveloper = objectMapper.readValue(jsonString, Developer.class);
+
+            developerService.addDeveloper(newDeveloper);
+        } catch (JsonProcessingException e) {
+            LOGGER.info("New developer is added!");
```

Comment:
1. why you call info logger in the catch block?
2. You got error on creation and response with the message `New developer is added!`...